media-sound/ocp/ocp-0.1.22~git.recipe

```diff
@@ -1,70 +1,83 @@
 SUMMARY="Open Cubic Player"
-DESCRIPTION="
-ocp - Open Cubic Player, a music player ported from DOS.
-"
+DESCRIPTION="ocp - Open Cubic Player, a music player ported from DOS."
 HOMEPAGE="http://stian.cubic.org/project-ocp.php"
-SOURCE_URI="git://git.code.sf.net/p/opencubicplayer/code#f97d87f960624de013826e9c2970404f90eef211"
-REVISION="1"
+COPYRIGHT="1994-2016 Niklas Beisert, Stian Skjelstad and others"
 LICENSE="GNU GPL v2"
-COPYRIGHT="1994-2012 Niklas Beisert, Stian Skjelstad and others"
-##		media-libs/flac >= 1.2.1"
-## ncurses zlib ... cf. ocp.spec
+REVISION="1"
+gitRevision="835bf803f590512848282bbf74986ebfaec17da3"
+SOURCE_URI="https://sourceforge.net/code-snapshots/git/o/op/opencubicplayer/code.git/opencubicplayer-code-$gitRevision.zip"
```

Comment:
Official release tarballs are available at https://sourceforge.net/projects/opencubicplayer/files/ocp-0.1.21/

Reply:
Thats from 2011, the git source have changes from 2016. If one want, can change it to the release, i don't see the point right now.